### PR TITLE
Opt into v3 alert component

### LINKED ISSUE #74189: web components v3 migration: Alert (opt in)
## Tasks

- [ ]  Explicitly **_opt in_** to v3 for all uses of the Alert web component in 526. This involves adding the `uswds=true` prop for all uses of the component
- [ ]  Investigate cases of alert styling, but not using the alert component (e.g. on the Introduction page). Consider switching to the Alert component for consistent styling and maintainability
- [ ]  Update any tests as needed
- [ ]  Provide a list of all affected pages and screenshots which will be helpful for manual testing

Fyi
- looks like there are 12 uses of the web component throughout 526
- there are styling differences between v1 and v3, including border and background color changes

### PR DESCRIPTION
**Pull request by christinec-fftc**
## Summary
-Opt in to v3 styling for alert components in 526
-Minor content updates to align with [content guidance](https://design.va.gov/content-style-guide/links#linking-to-external-sites) 
-Two changes to convert from markup + utility styling to v3 component for ui/ux consistency and future maintainability

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#74189

## Testing done
-Manual, visual verification. Updated unit test and verified e2e tests still pass

## Screenshots & Manual testing
Before and after screenshots. (before on the left, after on the right)
1. MissingServices.jsx
	This can be reproduced by logging in locally with mock user 1
image

2. adaptive-benefits > adaptiveBenefits.jsx
On the adpative-benefits page, select
	Do you need help buying or modifying your car? -> Yes
	Have you ever been granted an automobile allowance? -> Yes
image

3. new-disabilities/add > addDisabilities.jsx > newOnlyAlert
On the new-disabilities page, do not add any new conditions. Click Continue button
image

4. new-disabilities/add > addDisabilities.jsx > increaseAndNewAlert
For claim type, make sure to set both new and CFI.
Couldn't find a local test user with RD, so couldn't reproduce this manually. e2e produces the validation error, by setting the following data in `src/applications/disability-benefits/all-claims/tests/fixtures/data/maximal-test.json`, and run `all-claims.cypress.spec.js`
```	    
"view:claimType": {
   "view:claimingIncrease": true,
  "view:claimingNew": true
},
...		
newDisabilities: []
```
image

5. disabilities/rated-disabilities > ratedDisabilities.jsx > ratedDisabilitiesAlert
repro similar to item 4 above

```
"view:claimType": {
   "view:claimingIncrease": true,
   "view:claimingNew": false
 },
```

for each item in `ratedDisabilities[]`, set `"view:selected": false`
image

6. /supporting-evidence/additional-evidence 
Start bdd claim. On the /supporting-evidence/evidence-types-bdd page, select 'Yes' and check 'Required Separation Health Assessment - Part A Self-Assessment'    
image

7. confirmation
Displayed after successful submit of BDD claim
image

8. supporting-evidence/evidence-types-bdd
Start bdd claim. In the '/supporting-evidence/evidence-types-bdd page select 'No, I will submit more information later'
image

9. how-claims-exams-work
no special instructions, just navigate to the page
image

10. confirmation-page (retryable error)
Submit a claim. This warning happens often on local and staging.
image

11. confirmation-page (error)
not sure how to repro this one 

12. itf wrapper
Start a new claim or come back to an in progress application
image

13. /supporting-evidence/service-treatment-records
Start a BDD claim. On the service-treatment-records page, select 'No, I will submit them later'
image

14. introduction
No need to sign in, just go to the intro page. This one includes a change from markup + utility classes to using the `va-alert` component. This is using the `slim` version of the alert component since there is already another alert component higher up on the page.
image

15. introduction
Start bdd flow. Change from markup + utility classes to `va-featured-content` component
image

## What areas of the site does it impact?
526ez

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by aurora-a-k-a-lightning — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx [hunk 1]
```diff
@@ -61,8 +61,7 @@ export const claimExamsFAQ = (
             rel="noreferrer"
           >
             {' '}
-            contact an Overseas Military Services Coordinator (opens in a new
-            tab)
+            contact an Overseas Military Services Coordinator (opens in new tab)
```
Comment (tommasina-va) [inline src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx:64]: I think we lost the "a" here :)

Comment (tommasina-va) [reply src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx:64]: Scratch that! I see you're removing all the 'a's. 👍🏻

**Review by tommasina-va — APPROVED** [review]
Nice work :)

File: src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx (hunk 1 above)
Comment (christinec-fftc) [reply src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx:64]: @tommasina-va Thanks for asking. I noticed the content guidance doesn't have an 'a', so removed it 
image

**Pull request merged**